src/instrument/plans/detector_prep.py

```diff
@@ -35,25 +39,38 @@ def prepare_count(
     image_mode = "Multiple" if n_images > 1 else "Single"
 
     yield from bps.mv(
-        plugin.enable, 1
-        plugin.create_directory, create_directory,
+        plugin.enable,
+        1,
+        plugin.create_directory,
+        create_directory,
     )
     # MUST set create_directory BEFORE file_path
 
     yield from bps.mv(
-        plugin.file_path, file_path,
-        plugin.file_template, file_template,
+        plugin.file_path,
+        file_path,
+        plugin.file_template,
+        file_template,
     )
 
     # NOW, set the other components
     yield from bps.mv(
```

Comment:
Wrap with `# fmt: off` &  `# fmt: on` to preserve the meaning (and ignore standard styling).